main/getting-started/intro-zoe.md

```diff
@@ -0,0 +1,279 @@
+
+
+Why the need to parameterize a contract with terms? Well, the minimum raise ]should be quite different if an auction is for a used Harry Potter paperback vs. a signed first British hardback edition of *Harry Potter and the Philosopher’s Stone*]. The `terms` value for the used paperback auction would be `{ Currency: 'USDollar', MinRaise: 1 }`. For the first edition auction, `terms`’ value would be  `{ Currency: 'USDollar', MinRaise: 500)`. With the different `terms` values, there are now contract instances customized for the items being sold in each one.
+
+## Invites
+
+The `E(zoe).makeInstance()` call returns an *invite* to that contract instance. Invites are [*ERTP payment objects*](https://agoric.com/documentation/getting-started/ertp-introduction.html#creating-assets-with-ertp) that let their holder interact with that specific contract instance. If there are ten instances of a contract running and you hold an invite to, say, the earliest created one, that’s the only one the invite works with. It doesn’t work with any of the nine later created contract instances. 
+
+
+And while whoever instantiated the contract gets the initial invite to that instantiation, as with all invites, they can transfer it to another party before it’s used. Who created or first held an invite doesn’t matter. Whoever possesses an invite has control and use of it. 
+
+
+Users obtain invites to a contract instance in one (or more) of three ways:
+- Instantiate a contract, which returns an invite object for that contract instance.
+- By receiving an invite from someone who already had it. 
+  - Note: Finding other parties to invite is out of Zoe or the contract’s scope in most cases. You have to either already know of potential other parties, for example, from past experience trading with them, or communicate somewhere outside the contract to find them.
+- By the contract instance or someone with the authority to issue invites for this contract instantiation making open knowledge what’s needed to get a valid invite. For example, an open exchange contract with an order book might welcome everyone who wants to participate, and so would make public how to get an invite to it.
+
+
+*Invites* are ERTP *payments*, so the same principles apply to them as apply to other ERTP payments. You can send the same payment or invite to multiple parties...but when a party claims the payment, that payment is dead. The claimant gets a new payment they know was issued by the desired issuer, and which they have exclusive access to. 
```

Comment:
maybe instead: "but when a party claims the payment after someone else, that payment is dead"